nixos/maintainers/scripts/ec2/create-amis.sh

```diff
@@ -248,7 +273,7 @@ upload_image() {
         write_state "$state_key" ami_id "$ami_id"
     fi
 
-    make_image_public "$region" "$ami_id"
+    [[ -v PRIVATE ]] || make_image_public "$region" "$ami_id"
```

Comment:
what's `-v` do for test? I don't see it in the manpage

Reply:
from the bash man page:
```
-v varname
    True if the shell variable varname is set (has been assigned a value). 
```

Reply:
> Please run this through shellcheck

So after trying a shell check, it seems to be a bit too pendantic on shell escaping. For the regions variable, for example, it wants to be escape the values, but I actually _want_ this particular var to be split on word.

Reply:
ah, not sure why i couldn't find it